pkg/agent/baker.go

```diff
@@ -548,12 +548,18 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			if profile != nil && profile.KubernetesConfig != nil && profile.KubernetesConfig.ContainerRuntimeConfig != nil && profile.KubernetesConfig.ContainerRuntimeConfig[datamodel.ContainerDataDirKey] != "" {
 				return true
 			}
+			if profile.KubeletDiskType == datamodel.TempDisk {
```

Comment:
the thinking here is we'll remove ContainerRuntimeConfig. 

it ended up not being very useful, and actually we can't set arbitrary paths for this field anyway -- it needs to be predictable for monitoring and mounting symlinks properly into containers reading docker logs.

Reply:
there are no active users of the toggle anyway since we pushed ephemeral os :)

Reply:
(ContainerRuntimeConfig is how this value was propagated from the toggle)